homework/02LessonTwo.js

```diff
@@ -26,6 +26,12 @@ var alwaysTrue = function () {
 // argument1 is equal to argument2
 
 function equals(argument1, argument2) {
+  if(argument1 === argument2){
+    return true;
+  }
+  else{
+    return false;
+  }
 }
```

Comment:
This is good. Note that you can also do this:

```javascript
function equals(argument1, argument2) {
  return argument1 === argument2;
}
```

Because `argument1 === argument2` evaluates to the correct boolean value.